tools/windows-node-installer/pkg/client/client.go

```diff
@@ -0,0 +1,28 @@
+package client
+
+import (
+	"fmt"
+
+	client "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/openshift"
+	"k8s.io/client-go/tools/clientcmd"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+// log is the global logger for the client package. logf uses logr generic interface. Each log record produced
+// by this logger will have an identifier containing `client` tag.
+var log = logf.Log.WithName("client")
+
+// GetOpenShiftClient uses kubeconfig to create a client for existing OpenShift cluster and returns it or an error.
+func GetOpenShiftClient(kubeConfigPath string) (*openshift.Client, error) {
```

Comment:
Given the way this will be used, `client.GetOpenShiftClient()`, I suggest renaming it to `GetOpenShift()`. But @ravisantoshgudimetla should weigh in too.